modules/sno-du-configuring-sriov.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-configuring-sriov_{context}"]
+= Configuring single root I/O virtualization (SR-IOV)
+
+SR-IOV is commonly used to enable the fronthaul and the midhaul networks.
+
+Procedure
+
+* Use the following configuration to configure SRIOV on a single node distributed unit (DU). Note that the first custom resource (CR) is required. The following CRs are examples.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovOperatorConfig
+metadata:
+  name: default
+  namespace: openshift-sriov-network-operator
+spec:
+  configDaemonNodeSelector:
+    node-role.kubernetes.io/master: ""
+  disableDrain: true
+  enableInjector: true
+  enableOperatorWebhook: true
+---
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-nw-du-mh
+  namespace: openshift-sriov-network-operator
+spec:
+  networkNamespace: openshift-sriov-network-operator
+  resourceName: du_mh
+  vlan: 150 <1>
+---
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: sriov-nnp-du-mh
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: vfio-pci <2>
+  isRdma: false
+  nicSelector:
+    pfNames:
+      - ens7f0 <3>
+  nodeSelector:
+    node-role.kubernetes.io/master: ""
+  numVfs: 8 <4>
+  priority: 10
+  resourceName: du_mh
+---
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-nw-du-fh
+  namespace: openshift-sriov-network-operator
+spec:
+  networkNamespace: openshift-sriov-network-operator
+  resourceName: du_fh
+  vlan: 140 <5>
+---
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: sriov-nnp-du-fh
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: netdevice <6>
+  isRdma: true
+  nicSelector:
+    pfNames:
+      - ens5f0 <7>
+  nodeSelector:
+    node-role.kubernetes.io/master: ""
+  numVfs: 8 <8>
+  priority: 10
+  resourceName: du_fh
+----
+
+<1> The VLAN for the midhaul network.
+<2> You can select either `vfio-pci` or `netdevice`, as needed.
```

Comment:
Make the call-outs consistent throughout. Some start with verb, others don't (fwiw, I usually uses Specifies...)

I don't think you need the _You can_ for <2> and <6>

Reply:
changed.